### ISSUE TITLE
Fan 3 showing 0 RPM

### ISSUE THREAD

**Issue by Nomar383** [AndrewX192/lenovo-sa120-fanspeed-utility#18]
I don't have any hardware error lights on, but the script always says Fan 3 is at 0 RPM. Which specific fan is that? I may try to replace one or all the fans, but I'm trying to narrow down which one might have failed.

**Comment by CannyGent** [comment]
My fan 3 is at 0 RPM too - did you figure this out?

**Comment by Nomar383** [comment]
Nope, it still always shows 0. I’ve looked at the back of the unit while
it’s running and it’s hard to tell if any of the fans are actually NOT
spinning.

-Kalen

**Comment by timnovinger** [comment]
I'm having the same issue (though I haven't yet physically checked to see if the fan is spinning or not). Here's what I see using the raw `sg_ses` command though.

https://sg.danny.cz/sg/sg_ses.html

<img width="504" alt="Screenshot 2024-01-30 at 13 36 12" src="https://github.com/AndrewX192/lenovo-sa120-fanspeed-utility/assets/28424/aa5a3703-b9c6-49eb-b9cb-c4ac6edce577">

**Comment by timnovinger** [comment]
I only have a single PSU in my SA120. So I'm guessing that's why there's only 5 fans reporting instead of 6. Is this the case for you all as well?

<img width="842" alt="Screenshot 2024-01-30 at 13 46 09" src="https://github.com/AndrewX192/lenovo-sa120-fanspeed-utility/assets/28424/0d57572b-5f67-494a-a05a-a42399647df3">

<img width="442" alt="Screenshot 2024-01-30 at 13 42 53" src="https://github.com/AndrewX192/lenovo-sa120-fanspeed-utility/assets/28424/a0cc170a-8093-4655-aa80-5707d0f595a9">

**Comment by Nomar383** [comment]
I have both PSUs and both are plugged in. I do think that the 6 fans are
one per PSU and two per fan assembly though. So it’s possible my second PSU
has a bad fan and you don’t have a second PSU at all

**Comment by CannyGent** [comment]
> I only have a single PSU in my SA120. So I'm guessing that's why there's only 5 fans reporting instead of 6. Is this the case for you all as well?

Good point - single PSU here so presumably that is the reading on mine. Thanks.
